src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Invoice/InvoicePaymentDto.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class InvoicePaymentDto
+    {
+        public int Amount;

```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)